nengo_viz/components/netgraph.py

```diff
@@ -42,12 +153,16 @@ def get_parents(self, uid):
         return parents
 
     def update_client(self, client):
+        self.check_for_reload()
```

Comment:
It's probably close to 50 times per second.  I think I'll switch this so it only calls check_for_reload every half-second or so, which should be sufficient.